help/visualize/viewers/sunburst.md

```diff
@@ -2,90 +2,75 @@
 title: "Sunburst"
 ---
 
-A [sunburst diagram](https://en.wikipedia.org/wiki/Pie_chart#Ring),
-also referred to as a Ring Chart, Sunburst Chart, or Multi-level Pie Chart,
-is a visual representation used to illustrate hierarchical categories as a series of rings.
+A [sunburst diagram](https://en.wikipedia.org/wiki/Pie_chart#Ring) organizes categories into rings, revealing a hierarchy. It is also known as a Ring Chart or Multi-Level Pie Chart.
 
-The central circle represents the root node,
-and the categories within the hierarchy extend outward in rings.
+## Structure
 
-Each ring corresponds to a specific attribute or column.
-For example, in the picture below, the inner circle represents the "Sex" column with categories "Male" and "Female".
-The next ring represents the boolean "Control" status, indicating whether an individual is in the control group or not.
-Finally, the outer circle represents the "Race" column.
+The middle circle represents the root node. Categories spread out in rings around it.
+
+Every ring stands for a particular trait or column.
+
+We split and cut the rings based on their hierarchy in relation to the parent slice. Each sector on the outer rings displays a combination from the inner rings.
```

Comment:
why 'split and cut'? Can we use just 'split'?

Reply:
Not very clear for me what happens here... Do we split the rings based on categories distribution within each parent slice?

Reply:
Yes, that exactly what was meant. But I tried to specify that there is an hierarchy. Some suggestions on how to make this sentence better?